frog/imports/client/Wiki/index.js

```diff
@@ -182,6 +184,13 @@ class WikiComp extends React.Component<WikiCompPropsT, WikiCompStateT> {
       );
     }
 
+    if (
+      this.wikiDoc.data.settings?.locked &&
+      !this.wikiDoc.data.owners.find(x => x === Meteor.userId())
+    ) {
+      this.props.showModal(<LockedModal />);
+    }
+
```

Comment:
Nit: a quick comment explaining the context behind this conditional would be useful